geps/gep-3779/index.md

```diff
@@ -0,0 +1,68 @@
+# GEP-3779: Identity Based Authz for East-West Traffic
```

Comment:
Concurrently the admin network policy is looking into adding service account based policies with the possibility to include mesh and L7 cases. We will need cross-project collaboration here to make sure we have a cohesive story

Reply:
lets move conversation to https://github.com/kubernetes-sigs/gateway-api/pull/3822. @aryan16 and I are collaborating there.